js/area/signin/signin.js

```diff
@@ -217,7 +217,9 @@ SignInComponent.prototype.reset = function () {
 		.removeClass('signin--error')
 		.find('.signin__hint').remove();
 
-	this.$html.find('input:not([name="username"])').trigger('blur');
+	if (this.$html.find('.signin-container input:not([name="username"])').length) {
```

Comment:
Why are there changes to the signin component here?

Reply:
That method (`SignInComponent.reset`) was bound to a `ESC` keyup event on `document` and was triggering every time `ESC` was pressed on any input, which interfered with focus management on the date picker (and potentially other things that listen for `ESC`). I've tried to keep the fix as simple as possible by only triggering blur on inputs within the `signin-container`.